packages/dds/tree/src/test/feature-libraries/objectForest.spec.ts

```diff
@@ -7,26 +7,34 @@ import { strict as assert } from "node:assert";
 
 import { validateAssertionError } from "@fluidframework/test-runtime-utils/internal";
 
-import { type FieldKey, moveToDetachedField, rootFieldKey } from "../../core/index.js";
+import {
+	type FieldKey,
+	moveToDetachedField,
+	rootFieldKey,
+	TreeStoredSchemaRepository,
+} from "../../core/index.js";
 import { cursorForMapTreeNode, initializeForest } from "../../feature-libraries/index.js";
 // Allow importing from this specific file which is being tested:
 /* eslint-disable-next-line import/no-internal-modules */
 import { buildForest } from "../../feature-libraries/object-forest/index.js";
-import { type JsonCompatible, brand } from "../../util/index.js";
+import { Breakable, type JsonCompatible, brand } from "../../util/index.js";
 import { testForest } from "../forestTestSuite.js";
-import { testIdCompressor, testRevisionTagCodec } from "../utils.js";
+import { testIdCompressor, testRevisionTagCodec, validateUsageError } from "../utils.js";
 import { fieldJsonCursor } from "../json/index.js";
+import { toStoredSchema, SchemaFactory } from "../../simple-tree/index.js";
 
 describe("object-forest", () => {
 	describe("forest suite", () => {
 		testForest({
-			factory: (schema) => buildForest(),
+			factory: (schema) => buildForest(new Breakable("testForest")),
 		});
 	});
 
-	describe("forest suite additional assertions", () => {
+	// TODO: the forest test suite should be able to run with the additional assertions.
```

Comment:
Should we create a follow-up task to fix these?

Reply:
Fixing those tests looked painful and low value.

I don't think this is in the top ~30% urgency wise of our 452 TODOs in the tree source, so I don't think its worth filing an item for. We still have full coverage from the version above, so this isn't really a regression (this batch of tests with validation on was only added in my PR I just made with cleanups to support this one).

Reply:
Should those tests just be removed then? Generally, I am of the opinion that `.skip`s should be short-term mitigation, and we should avoid letting the set of skipped tests grow, since it adds maintenance burden without the benefit of test coverage.

Reply:
Maintaining this invocation of testForest should be relatively trivial, and I think is the most maintainable way to document how this would be tested if the TODO were implemented.

If this skip had a lot of code inside of it, instead of a single method call, then I'd suggest removing it (or prioritizing fixing it), but as is I don't really see a downside to keeping it, though maybe I'm missing something.